static/index.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html lang="en-gb">
+<head>
+	<meta charset="utf-8">
+
+	<title>Aleph faucet</title>
+	<meta name="description" content="Get free tokens for the Aleph Zero testnet.">
+	<link rel="stylesheet" type="text/css" href="main.css" />
+</head>
+<body>
+	<h1> Faucet </h1>
+
```

Comment:
Would be good to put a description like: "Provide your address to obtain free tokens."

Reply:
It should also have a proper picture with the Aleph logo working as a showerhead, but this was supposed to be just a weak prototype. D:

Anyway, added as suggested.